src/background/services/wallet/handlers/importLedger.ts

```diff
@@ -68,6 +70,16 @@ export class ImportLedgerHandler implements HandlerType {
       };
     }
 
+    if (onlyCheckWalletIsExist) {
+      return {
+        ...request,
+        result: {
+          id: '0',
+          type: secretType,
+        },
```

Comment:
Let's try to preserve the same response type as the success message normally has.
```suggestion
        result: {
          id: '0',
          name,
          type: secretType,
        },
```